db/view/view.cc

```diff
@@ -1282,7 +1282,10 @@ future<> mutate_MV(
         auto view_token = dht::get_token(*mut.s, mut.fm.key());
         auto& keyspace_name = mut.s->ks_name();
         auto target_endpoint = get_view_natural_endpoint(keyspace_name, base_token, view_token);
-        auto remote_endpoints = service::get_local_storage_proxy().get_token_metadata_ptr()->pending_endpoints_for(view_token, keyspace_name);
+        const auto& db = service::get_local_storage_proxy().local_db();
+        const auto& ks = db.find_keyspace(keyspace_name);
```

Comment:
Idea: give storage_proxy its own keyspace type and storage and avoid the need to access database for replication information.

Reply:
makes sense.  need to see how this work out wit respect to the mechanics of creating such a keyspace.
(and out of scope of this PR :)